packages/cxl-ui/src/components/cxl-star-rating.js

```diff
@@ -0,0 +1,64 @@
+import '@conversionxl/cxl-lumo-styles';
+import { IronStarRating } from '@cwmr/iron-star-rating';
+
+export class CxlStarRating extends IronStarRating {
+  static get is() {
+    return 'cxl-star-rating';
+  }
+
+  static get properties() {
+    return {
+      value: {
+        type: Number,
+        value: 0,
+        notify: true,
+        observer: '_valueChanged',
+      },
+      icon: {
+        type: String,
+        value: 'icons:star',
+      },
+      disableAutoUpdate: {
+        type: Boolean,
+        value: false,
+      },
+      readonly: {
+        type: Boolean,
+        value: false,
+        reflectToAttribute: true,
+      },
+    };
+  }
```

Comment:
Why do we have to repeat `properties()` method from parent class - is it not inherited automatically?

Reply:
Good point, I forgot to remove it, I was adding my own property there, and forgot that I dropped it.
Fixed in: https://github.com/conversionxl/aybolit/pull/88/commits/cc8ac24026c5c5dfac572e3851d694ad528355eb